src/pages/resources/recipes/create-ui.md

```diff
@@ -0,0 +1,159 @@
+   1. You need to `import` individual comps before using them
+   2. You can debug SWC like a regular Web Component. 
+
+
+
+That said, you can mix and match these three offerings to suit your use case. For example, the following will work seamlessly
+
+
+```HTML
+<form> <!-- plain HTML element -->
+   <sp-banner> <!-- Spectrum Web Component -->
+     <div slot="header">Header text</div>
+     <div slot="content">Content of the banner</div>
+  </sp-banner>
+  <sp-button variant="primary">I'm a button</sp-button> <!-- Spectrum widget -->
+</form>
+```
+
+### What's best for you?
+
+
+The choice you make depends on your use case and the stage of development you are in.
+
+
+If you are looking for a ready-made Adobe Spectrum-styled UI or are just getting started with programming your UI, we recommend you start your journey with SWC. Fall back to Spectrum UXP widgets for the comps that are not available yet. And use plain HTML elements when they are not available in either.
+
+
+However, if you prefer vanilla HTML and don't mind undertaking the efforts of styling, use the HTML elements along with your custom stylesheet. In fact, you can build your own custom tags using Web Components. Make sure you read the next section of [CSS Styling](./styling.md)
+
+## Additional notes
+
+While writing a simple JS/HTML/CSS plugin is possible, those who are building plugins that have internal state management, or more complicated UI may benefit from commonly used UI libraries. Below examples are not necessarily only ones you can use, but can serve as good starters if you already are familiar with a library.
+
+We do not suggest that you use any particular library for your plugins, but suggest that you read about them and decide for yourself what will work the best for you.
+
+### React
+
+[React.js](https://reactjs.org/) is one of the most used UI libraries. It's declarative and component based, helping you break your UI into reusable components. 
+
+See [ui-react-starter](https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/tree/main/ui-react-starter) for the sample project. // TODO update link
+
+### Vue
+
+[Vue.js](https://vuejs.org/) is another declarative UI library that's easy to learn. 
+
+See [ui-vue-starter](https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/tree/main/ui-vue-starter) for the sample project.  // TODO update link
+
+### Svelte
+
+[Svelte](https://svelte.dev/) is different from the other two libraries, in the sense that it compiles your code to a simple JS application, shrinking it's size.
+
+See [ui-svelte-starter](https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/tree/main/ui-svelte-starter) for the sample project.  // TODO update link
+
+
+
+## Reference material
+- [Spectrum Web Components](swc/index.md) and [sample plugin](https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/tree/main/swc-uxp-starter) //TODO link to Id kitchen sink app
```

Comment:
@kareemmyk

Reply:
@padmkris123 ... 
Let's add this link [ui-kitchen-sink](https://git.corp.adobe.com/torq/uxp-indesign-samples/tree/main/ui-kitchen-sink) . However, I notice that few components are broken in InDesign. For example the following components are not working the same way as Photoshop

![image](https://github.com/AdobeDocs/uxp-indesign/assets/34368858/2eb9cde6-d4bd-4f4b-9e3d-c38c6a6f46b1)

Reply:
@kareemmyk  These issues should be captured in 'Known Issues'. Please create a PR to update this page https://github.com/AdobeDocs/uxp-indesign/blob/new-layout/src/pages/known-issues/index.md